.travis.yml

```diff
@@ -138,16 +138,16 @@ jobs:
         #CI - docker
       #CI env: CMAKE_CC="clang-3.8" CMAKE_CXX="clang++-3.8" CONAN_COMPILER="clang" CONAN_COMPILER_VERSION="3.8" TARGET=linux_clang docker_tag="ubuntu-zesty-clang038" NIGHTLY="conan"
 
-    #- os: osx
-      #sudo: false
-      #osx_image: xcode8
-      #language: cpp
-      #compiler: clang
-      #env: docker_tag="NODOCKER" compiler=clang TARGET="osx" CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=clang++-3.9 -DCMAKE_C_COMPILER=clang-3.9 -DBUILD_ALL=1"
+    #CI - os: osx
```

Comment:
someday I'd like to split travis.yml again in two files to make this pipeline a bit more intuitively understandable :D ive no clue atm since "CI" is pretty ambigous
not now though